<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="BibiButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>打开报警器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>网络状态：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BibiButtonoff">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭报警器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OutButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>外出模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>回家模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="NetworkLable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>连接中...</string>
   </property>
  </widget>
  <widget class="QLabel" name="ComLable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>连接中...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>串口状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ModeLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>回家模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>当前模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>当前温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="TemLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0°</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>当前湿度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="HumLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="PMLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PM2.5：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>系统时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeLable">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QToolButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
